addon/index.js

```diff
@@ -24,26 +28,19 @@ const mod = new PageMod({
   onAttach: setupApp
 });
 
-if (!store.experiments) {
-  store.experiments = {};
-}
-
-if (!store.availableExperiments) {
-  store.availableExperiments = [];
+if (!store.clientUUID) {
+  store.clientUUID = generateUUID();
 }
 
 require('sdk/system/unload').when(function(reason) {
   if (reason === 'uninstall') {
     app.send('addon-self:uninstalled');
-    if (store.experiments) {
-      for (let ex in store.experiments) { // eslint-disable-line prefer-const
-        if (store.experiments.hasOwnProperty(ex)) {
-          uninstall({addon_id: ex});
-        }
+    if (store.installedAddons) {
+      for (let id of store.installedAddons) { // eslint-disable-line prefer-const
```

Comment:
Should we just do `for (const id of store.installedAddons) {` (since `id` never changes) and then we can delete that ESLint rule override?